FiveCalls/FiveCalls/AppDelegate.swift

```diff
@@ -41,6 +42,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
 
         oneSignalStartup(launchOptions: launchOptions)
+        Mixpanel.sharedInstance(withToken: "776fce75f7e3ddfbb13b615dcb94ff95")
```

Comment:
Do we already have a place to store this stuff? I don't like committing tokens in the source when I can avoid it.

Reply:
Info.plist seems like what we've been using. I'll pop it in there.